src/github/tests/test_utils.rs

```diff
@@ -254,23 +431,46 @@ impl GithubRead for GithubMock {
             .collect())
     }
 
-    fn repo(&self, _org: &str, _repo: &str) -> anyhow::Result<Option<Repo>> {
-        todo!()
+    fn repo(&self, org: &str, repo: &str) -> anyhow::Result<Option<Repo>> {
+        assert_eq!(org, DEFAULT_ORG);
+        Ok(self.repos.get(repo).cloned())
     }
 
-    fn repo_teams(&self, _org: &str, _repo: &str) -> anyhow::Result<Vec<RepoTeam>> {
-        todo!()
+    fn repo_teams(&self, org: &str, repo: &str) -> anyhow::Result<Vec<RepoTeam>> {
+        assert_eq!(org, DEFAULT_ORG);
+        Ok(self
+            .repo_members
+            .get(repo)
+            .cloned()
+            .map(|(teams, _)| teams)
+            .unwrap_or_default())
     }
 
-    fn repo_collaborators(&self, _org: &str, _repo: &str) -> anyhow::Result<Vec<RepoUser>> {
-        todo!()
+    fn repo_collaborators(&self, org: &str, repo: &str) -> anyhow::Result<Vec<RepoUser>> {
+        assert_eq!(org, DEFAULT_ORG);
```

Comment:
Is the plan to remove this restriction soon? Or why else would we accept an `org` in the method instead of hardcoding it?

Reply:
I wouldn't say soon :sweat_smile: The sync code is more or less prepared for multiple repos, but we currently have only one, and using a single repo made the tests much simpler. I tried to prepare them at least a little bit for multiple orgs in the future, but since we only use now, I thought that it is better to just keep it simple for now.